pkg/util/util_test.go

```diff
@@ -125,23 +126,144 @@ func TestConvertTagsToMap(t *testing.T) {
 }
 
 func TestMakeDir(t *testing.T) {
-	targetTest := "./target_test"
-	//Successfully create directory
-	err := MakeDir(targetTest)
-	assert.NoError(t, err)
-
-	// Remove the directory created
-	err = os.RemoveAll(targetTest)
-	assert.NoError(t, err)
+	testCases := []struct {
+		desc          string
+		setup         func()
+		targetDir     string
+		expectedError bool
+		cleanup       func()
+	}{
+		{
+			desc:          "should create directory",
+			targetDir:     "./target_test",
+			expectedError: false,
+		},
+		{
+			desc: "should not return error if path already exists",
+			setup: func() {
+				err := os.Mkdir("./target_test", os.FileMode(0755))
+				assert.NoError(t, err, "Failed in setup: %v", err)
+			},
+			targetDir:     "./target_test",
+			expectedError: false,
+		},
```

Comment:
You should remove the directory after the test case has run just like in `TestMakeFile`.

Reply:
The removal should be handled by the following block in this case:

https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/743802637524d7759b70c08979211a12b7c0bb37/pkg/util/util_test.go#L173-L177

Reply:
Oh missed that. Thanks for pointing it out!